src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1054,7 +1054,18 @@ function _vcat(dfs::AbstractVector{<:AbstractDataFrame};
     # List of symbols present in all dataframes
     intersectunique = intersect(uniqueheaders...)
 
-    if cols === :equal
+    if cols === :orderequal
+        header = unionunique
+        if length(uniqueheaders) > 1
+            throw(ArgumentError("all data frames to have the same column names " *
+                                "and in the same order"))
+        end
+    elseif cols === :setequal || cols === :equal
+        if cols === :equal
+            Base.depwarn("`cols` value eqaual to `:equal` is deprecated." *
```

Comment:
```suggestion
            Base.depwarn("`cols=:equal` is deprecated." *
```

Reply:
Maybe it should be noted that the default changes. Otherwise people don't understand where this warning comes from since they didn't use `:equal` anywhere

Reply:
No, the default is now `:setequal` so people will only get the warning if they pass `:equal` explicitly (i.e. never :-).

Reply:
@nalimilan hit the 🎯 here 😄.